packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -88,26 +88,20 @@ const deployImplementation = async (
   return contract
 }
 
-const deployProxy = async (contractName: string, addresses: ContractAddresses, dryRun: boolean) => {
-  // Explicitly forbid upgrading to a new Governance proxy contract.
-  // Upgrading to a new Governance proxy contract would require ownership of all
-  // contracts to be moved to the new governance contract, possibly including contracts
-  // deployed in this script.
-  // Because this depends on ordering (i.e. was the new GovernanceProxy deployed
-  // before or after other contracts in this script?), and that ordering is not being
-  // checked, fail if there are storage incompatible changes to Governance.
-  if (contractName === 'Governance') {
```

Comment:
Do we not need this check anymore?

Reply:
Ah, see it's been moved lower. Should it remain here to keep the Proxy deployment logic in the `deployProxy` function?

Reply:
for some reason I thought this was easier than passing around the from address
will revert